sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_operations/_operations.py

```diff
@@ -0,0 +1,1420 @@
+        """
+        error_map = {
+            401: ClientAuthenticationError,
+            404: ResourceNotFoundError,
+            409: ResourceExistsError,
+            304: ResourceNotModifiedError,
+        }
+        error_map.update(kwargs.pop("error_map", {}) or {})
+
+        _headers = kwargs.pop("headers", {}) or {}
+        _params = kwargs.pop("params", {}) or {}
+
+        cls: ClsType[_models.GetLanguagesResult] = kwargs.pop("cls", None)
+
+        request = build_text_translation_get_languages_request(
+            client_trace_id=client_trace_id,
+            scope=scope,
+            accept_language=accept_language,
+            if_none_match=if_none_match,
+            api_version=self._config.api_version,
+            headers=_headers,
+            params=_params,
+        )
+        path_format_arguments = {
+            "Endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, "str", skip_quote=True),
+        }
+        request.url = self._client.format_url(request.url, **path_format_arguments)
+
+        _stream = False
+        pipeline_response: PipelineResponse = self._client._pipeline.run(  # pylint: disable=protected-access
+            request, stream=_stream, **kwargs
+        )
+
+        response = pipeline_response.http_response
+
+        if response.status_code not in [200]:
+            map_error(status_code=response.status_code, response=response, error_map=error_map)
+            error = _deserialize(_models.ErrorResponse, response.json())
+            raise HttpResponseError(response=response, model=error)
+
+        response_headers = {}
+        response_headers["X-RequestId"] = self._deserialize("str", response.headers.get("X-RequestId"))
+        response_headers["ETag"] = self._deserialize("str", response.headers.get("ETag"))
+
+        deserialized = _deserialize(_models.GetLanguagesResult, response.json())
+
+        if cls:
+            return cls(pipeline_response, deserialized, response_headers)  # type: ignore
+
+        return deserialized  # type: ignore
+
+    @overload
+    def translate(
+        self,
+        content: List[_models.InputTextItem],
+        *,
+        to: List[str],
+        client_trace_id: Optional[str] = None,
+        from_parameter: Optional[str] = None,
+        text_type: Optional[Union[str, _models.TextType]] = None,
+        category: Optional[str] = None,
+        profanity_action: Optional[Union[str, _models.ProfanityAction]] = None,
+        profanity_marker: Optional[Union[str, _models.ProfanityMarker]] = None,
+        include_alignment: Optional[bool] = None,
+        include_sentence_length: Optional[bool] = None,
+        suggested_from: Optional[str] = None,
+        from_script: Optional[str] = None,
+        to_script: Optional[str] = None,
+        allow_fallback: Optional[bool] = None,
+        content_type: str = "application/json",
+        **kwargs: Any
+    ) -> List[_models.TranslatedTextItem]:
```

Comment:
1. The position of input text is mapped to position in translated text. Each input element will have the same index in output.
2. Yes, we have limits set - they change from time to time (we changed it recently).

Reply:
Got it, thanks. Depending on the limit set, we might want to double-check this before GA as well. If it's expected to grow it might be better to represent this as ItemPaged (and async equivalent)